packages/in_app_purchase/in_app_purchase_ios/ios/Classes/FIAPReceiptManager.m

```diff
@@ -5,22 +5,41 @@
 #import "FIAPReceiptManager.h"
 #import <Flutter/Flutter.h>
 
+@interface FIAPReceiptManager ()
+
+- (NSData *)getReceiptData:(NSURL *)url error:(NSError **)error;
```

Comment:
Needs a declaration comment (Primarily explaining what it does, but secondarily mentioning that it exists to be stubbed out in tests.)